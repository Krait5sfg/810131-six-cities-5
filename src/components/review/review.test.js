import React from 'react';
import renderer from 'react-test-renderer';
import Review from './review';

const comment = {
  id: 1,
  avatar: `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/3.jpg`,
  name: `Isaac`,
  rating: 4,
  commentText: `Bathed in the nature. Completely unplugged. Unforgettable.`,
  isPro: true,
  date: new Date(`Tue Oct 27 2020 12:29:21 GMT+0300`),
};

it(`Review render correctly`, () => {
  const tree = renderer.create(<Review
    comment={comment}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
