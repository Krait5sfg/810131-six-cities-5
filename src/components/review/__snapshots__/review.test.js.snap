// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review render correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/3.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Isaac
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Bathed in the nature. Completely unplugged. Unforgettable.
    </p>
    <time
      className="reviews__time"
      dateTime="2020-10-27"
    >
      October 2020
    </time>
  </div>
</li>
`;
